core/storage/collection/gae_models_test.py

```diff
@@ -412,7 +441,10 @@ def test_revert_to_valid_version_is_successful(self):
         )
 
     def test_revert_to_version_with_invalid_status_is_successful(self):
-        broken_dict = dict(**self.original_dict)
+        # type: () -> None
+        # TODO(#13523): Any here will be removed when model_dict is converted
```

Comment:
I think a better wording will be - `Use of Any in the type annotation below will be removed when....`

Reply:
Done